examples/calc/calc.ts

```diff
@@ -1,6 +1,11 @@
 import {opsList, opsSymbolsList} from './ops';
 
-export function calc(argv: {[x: string]: string[]}): string {
+ 
+interface argvType {
```

Comment:
nit: You'd usually UpperCase the names of types.

```suggestion
interface ArgvType {
```

Reply:
(And then you can also call it `Argv` because its' no longer easily confused with a value of that type. :))

Reply:
Thanks! I knew I'd seen that somewhere and then couldn't find it.